packages/components/src/__future__/Tag/RemoveableTag.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { HTMLAttributes } from "react"
+import classNames from "classnames"
+import { OverrideClassName } from "~types/OverrideClassName"
+import { RemoveButton } from "./RemoveButton"
+import { Tag, TagProps } from "./Tag"
+import styles from "./RemoveableTag.module.scss"
+
+export type RemoveableTagProps = {
+  removeButtonProps: {
+    "aria-label": string
+    onClick: () => void
+  }
+} & TagProps &
```

Comment:
Icons are not available in this iteration.

```suggestion
Omit<Tag, "icon">
```